docs/source/jit.rst

```diff
@@ -531,13 +531,251 @@ Python-defined Constants
 Debugging
 ~~~~~~~~~
 
-Print things
+Interpreting Graphs
+    TorchScript uses a static single assignment (SSA) intermediate representation
+    (IR) to represent computation. The instructions in this format consist of
+    ATen operators and other primitive operators, including control flow
+    operators for loops and conditionals. As an example::
 
-Use ``USE_PYTHON=0`` to debug in normal python mode
+        @torch.jit.script
+        def foo(len : int) -> torch.Tensor:
```

Comment:
This is python3 only, should we mention that somewhere here or switch to python2 syntax?